slack_bolt/context/context.py

```diff
@@ -7,8 +7,17 @@
 from slack_bolt.context.base_context import BaseContext
 from slack_bolt.context.respond import Respond
 from slack_bolt.context.say import Say
+from slack_bolt.context.complete_success import CompleteSuccess
+from slack_bolt.context.complete_error import CompleteError
 from slack_bolt.util.utils import create_copy
 
+CLIENT: str = "client"
```

Comment:
I felt like these values were used many times in the file, to prevent typos I consolidated them in variables